.github/workflows/ansible-test-plugins.yml

```
@@ -50,142 +52,117 @@ jobs:
           - mysql
           - mariadb
         db_engine_version:
-          - 5.7.40
-          - 8.0.31
-          - 10.4.27
-          - 10.5.18
-          - 10.6.11
-        python:
-          - '3.8'
-          - '3.9'
-          - '3.10'
+          - '8.0.38'
+          - '8.4.1'
+          - '10.5.25'
+          - '10.11.8'
         connector_name:
           - pymysql
           - mysqlclient
         connector_version:
-          - 0.7.11
-          - 0.9.3
-          - 1.0.2
-          - 2.0.1
-          - 2.0.3
-          - 2.1.1
+          - '0.9.3'
+          - '1.0.2'
+          - '1.1.1'
+          - '2.0.1'
+          - '2.0.3'
+          - '2.1.1'
```

```suggestion
          - '0.9.3'
          - '1.0.2'
          - '1.1.1'
          - '2.0.1'
          - '2.0.3'
          - '2.1.1'
```
can we also apply same oldest-supported-and-newest strategy here?

---

Best way to install PyMySQL is to use the distribution's package manager.
The version depends on the will of the APT/RPM maintainers:
- Ubuntu 22.04 : 1.0.2
- RHEL8 : 0.10.1
- RHEL9 : 1.1.1
- Fedora39 : 1.1.0-3 and 1.1.1-1

So we can't say: "This 1.0.2 bug was fixed in 1.1.1, please update", if their distribution doesn't provide such package.

We won't save that many VMs anyway. And PyMySQL is not updated that often: https://pypi.org/project/PyMySQL/#history